src/rules/converters/no-implicit-dependencies.ts

```diff
@@ -0,0 +1,23 @@
+import { RuleConverter } from "../converter";
+
+export const convertNoImplicitDependencies: RuleConverter = tslintRule => {
+    const ruleArgs = [];
```

Comment:
This is causing a error on the CI test [as seen here](https://circleci.com/gh/typescript-eslint/tslint-to-eslint-config/2105?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link).

> _"TS7034: Variable 'ruleArgs' implicitly has type 'any[]' in some locations where its type cannot be determined."_

Reply:
Thanks, i'm looking into it